src/commands/realClick.ts

```diff
@@ -38,7 +46,7 @@ export async function realClick(
     ? { x: options.x, y: options.y } 
     : options.position;
 
-  const { x, y } = getCypressElementCoordinates(subject, position);
+  const { x, y } = getCypressElementCoordinates(subject, position, options.scrollBehavior);
```

Comment:
It would be nice if these commands would default the scrollBehavior to whatever value is set by `cypress.json`.

We could fetch that with `Cypress.config('scrollBehavior')`, but since the peerDeps for this library allow versions before Cypress 6.1.0, I was having trouble getting Typescript to cooperate with me.

I was developing with Cypress 5 locally, so the typescript compiler was complaining that "scrollBehavior" wasn't a valid argument for Cypress.config.

I'm not sure what to do about that- maybe we can check the version of Cypress that's installed, and only fetch the default config value if it's >= 6.1.0?

Reply:
Prior to 6.1.0, users could disable scrolling by passing `{ force: true }`. Not sure if we'd want to support the same parameter for earlier versions of Cypress?